pydantic_core/_types.py

```diff
@@ -121,6 +121,14 @@ class SetSchema(TypedDict):
     strict: NotRequired[bool]
 
 
+class FrozenSetSchema(TypedDict):
+    type: Literal['frozenset']
+    items: Schema
+    min_items: NotRequired[int]
+    max_items: NotRequired[int]
+    strict: NotRequired[bool]
```

Comment:
```suggestion
class FrozenSetSchema(TypedDict, total=False):
    type: Required[Literal['frozenset']]
    items: Schema
    min_items: int
    max_items: int
    strict: bool
```

Could fix set at the same time.